internal/helm/client_test.go

```diff
@@ -85,7 +85,7 @@ var _ = Describe("SynchronizedClient", func() {
 
 	When("installing or upgrading chart", func() {
 
-		It("should wait for releases in the same namespace", func() {
+		It("should not wait for different releases in the same namespace", func() {
```

Comment:
Should we add a test for `should wait for same releases in the same namespace` ?

Reply:
Oh my.. I thought that it was already covered by some other test but it's not the case.

Yes, absolutely. Let me add that.

Reply:
Fixed